src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,7 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
```

Comment:
Yes, but since you added `tinycolor.getOriginalInput()`, I think it changes expectations. I would expect that method to be called something like `toNormalizedString()`. I'll leave it up to you if you want to pursue that.